src/objects/zif_abapgit_intf.intf.abap

```diff
@@ -0,0 +1,16 @@
+INTERFACE zif_abapgit_intf
+  PUBLIC .
+
+  TYPES:
+    BEGIN OF ty_docu,
+      lines      TYPE tlinetab,
+      i18n_lines TYPE zif_abapgit_lang_definitions=>ty_i18n_lines,
+    END OF ty_docu.
+
+  TYPES:
+    BEGIN OF ty_intf,
+      vseointerf  TYPE vseointerf,
+      docu        TYPE ty_docu,
+      description TYPE zif_abapgit_oo_object_fnc=>ty_seocompotx_tt,
+    END OF ty_intf.
```

Comment:
suggest keeping the types in the class

Reply:
addressed in [da45326](https://github.com/abapGit/abapGit/pull/5668/commits/da4532670ed49f5c690a3b00ecac59b6b3024ef2)